src/main/java/org/spongepowered/common/registry/type/world/DifficultyRegistryModule.java

```diff
@@ -57,10 +61,10 @@
 
     @Override
     public void registerDefaults() {
-        this.difficultyMappings.put("peaceful", (Difficulty) (Object) EnumDifficulty.PEACEFUL);
-        this.difficultyMappings.put("easy", (Difficulty) (Object) EnumDifficulty.EASY);
-        this.difficultyMappings.put("normal", (Difficulty) (Object) EnumDifficulty.NORMAL);
-        this.difficultyMappings.put("hard", (Difficulty) (Object) EnumDifficulty.HARD);
+        this.difficultyMappings.put("minecraft:peaceful", (Difficulty) (Object) EnumDifficulty.PEACEFUL);
```

Comment:
You'll need to override [provideCatalogMap](https://github.com/SpongePowered/SpongeCommon/blob/master/src/main/java/org/spongepowered/common/registry/type/item/EnchantmentRegistryModule.java#L62).